common/men_at_arms_types/00_maa_types.txt

```diff
@@ -126,6 +127,35 @@ pikemen_unit = {
 	icon = pikemen
 }
 
+caster_unit = {
+	type = caster
+	
+	damage = 30
+	toughness = 9
+	pursuit = 5
+	screen = 0
+	
+	terrain_bonus = {
+		plains = { damage = 10 pursuit =10 }
+		drylands =  { damage = 10 pursuit =15 }
+		mountains = { damage = 14 pursuit = 14 }
+		hills = { damage = 8 toughness = 4 pursuit = 12}
+	}
```

Comment:
They get a huge pursuit bonus compared to their default. Their pursuit depends too much on terrain. It doesn't feel good.

Reply:
Not adressed.